{"name": "test_sub[except]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 5 == 6\n  +5\n  -6", "trace": "self = <ZhangNan01.test_jisuanqi.TestCalc object at 0x0000021E8B76C610>\nget_calc = <test_pytest.ZhangNan01.jisuanqi.Calculator object at 0x0000021E8B6F1190>, get_datas1 = [11, 6, 6]\n\n    @pytest.mark.run(order=3)\n    def test_sub(self, get_calc, get_datas1):\n        # 调用add函数,返回的结果保存在result里面\n        result = get_calc.sub(get_datas1[0], get_datas1[1])\n        # 判断result结果是否等于期望的值\n>       assert result == get_datas1[2]\nE       assert 5 == 6\nE         +5\nE         -6\n\ntest_jisuanqi.py:27: AssertionError"}, "parameters": [{"name": "get_datas1", "value": "[11, 6, 6]"}], "start": 1614180736012, "stop": 1614180736012, "uuid": "973dc914-a8a9-4251-8009-e980921380fc", "historyId": "4295b8c0999fa1965f0c98eb73051c9b", "testCaseId": "827663f2d2b259ebfc592899e6937789", "fullName": "ZhangNan01.test_jisuanqi.TestCalc#test_sub", "labels": [{"name": "tag", "value": "run(order=3)"}, {"name": "parentSuite", "value": "ZhangNan01"}, {"name": "suite", "value": "test_jisuanqi"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "DESKTOP-DQV1BM1"}, {"name": "thread", "value": "8460-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "ZhangNan01.test_jisuanqi"}]}